### PR TITLE
Add a compare script

### PR DESCRIPTION
**Pull request by senadir**
WIP, adds a compare script that compares frontend block sizes between two branches, how to run

```
./bin/compare/compare-script.sh release/2.9 release/3.0
```

will output the changes as a markdown table, still don't know how to silent other commands.
For it to work, it should be added to both branches, you can merge this branch into both to test.

### How to test

In order to test this, you need to have the code in both branches, so you can merge this PR into `release/2.9` and `release/3.0` to test it.

- run `./bin/compare/compare-script.sh release/2.9 release/3.0` in which `release/2.9` is the name of the previous branch and `release/3.0` is the name of the new branch.
- The script will output the result in the end.

| Bundle                                 | Release 2 9 | Release 3 0 | Diff     |
| -------------------------------------- | ----------- | ----------- | -------- |
| `./build/single-product-frontend.js`   | 144.42KB    | 101.94KB    | -42.48KB |
| `./build/all-products-frontend.js`     | 141.64KB    | 100KB       | -41.64KB |
| `./build/active-filters-frontend.js`   | 23.55KB     | 28.22KB     | 4.67KB   |
| `./build/cart-frontend.js`             | 226.35KB    | 230.7KB     | 4.35KB   |
| `./build/attribute-filter-frontend.js` | 58.59KB     | 62.65KB     | 4.06KB   |
| `./build/checkout-frontend.js`         | 287.69KB    | 289.46KB    | 1.77KB   |
| `./build/price-filter-frontend.js`     | 47.59KB     | 47.73KB     | 138B     |
| `./build/reviews-frontend.js`          | 30.81KB     | 30.88KB     | 78B      |

### CODE REVIEW

File: bin/compare/compare-script.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BEFORE_BRANCH=$1
+AFTER_BRANCH=$2
+
+git checkout $BEFORE_BRANCH > /dev/null
+npm install > /dev/null
```
Comment (nerrad) [inline bin/compare/compare-script.sh:7]: maybe use `npm ci` instead here? That ensures node_modules is clear and just `package-lock.json` is used which should prevent a new package-lock.json from being created.

File: package-lock.json [hunk 2]
```diff
@@ -41066,7 +41121,7 @@
 			}
 		},
 		"woocommerce": {
-			"version": "git+https://github.com/woocommerce/woocommerce.git#5213b05d57b0e0e703eaed3519694fa98a51516f",
+			"version": "git+https://github.com/woocommerce/woocommerce.git#16a289cefaf0c71d0ec4c9b8252ce5846973615c",
```
Comment (nerrad) [inline package-lock.json]: This hash bump is causing issues, don't know why it keeps changing :|. It'll be nice when we can stop including woocommerce as a npm package and just import `woocommerce/e2e-utils` instead.

**Comment by senadir** [comment]
Moving this to cooldown as I intend to handle it then.

**Review by nerrad — DISMISSED** [review]
Testing pull request approval github action.

**Review by nerrad — DISMISSED** [review]
Test Github PR approval action.

**Review by nerrad — DISMISSED** [review]

**Pull request closed without merging**